AmplifyPlugins/Auth/AWSAuthPlugin/Support/Utils/AuthCognitoSignedOutSessionHelper.swift

```diff
@@ -0,0 +1,161 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import AWSMobileClient
+import AWSPluginsCore
+
+struct AuthCognitoSignedOutSessionHelper {
+
+    /// Creates a signedOut session information with valid identityId and aws credentials.
+    /// - Parameters:
+    ///   - identityId: Valid identity id for the current signedOut session
+    ///   - awsCredentials: Valid AWS Credentials for the current signedOut session
```

Comment:
I thought we had agreed that the "signed out" state wouldn't return credentials?